src/input/text-area.directive.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, HostBinding } from "@angular/core";
+
+/**
+ * A directive for applying styling to a textarea element.
+ *
+ * Example:
+ *
+ * ```hmtl
```

Comment:
Typo?

Reply:
Good catch!

Reply:
fixed - was a copy-pasted typo from the input directive.